contexts/security-v3-unstable.jsonld

```diff
@@ -67,6 +67,69 @@
         }
       }
     },
+    "EcdsaSecp256k1RecoveryMethod2020": {
+      "@id": "https://w3id.org/security#EcdsaSecp256k1RecoveryMethod2020"
+    },
+    "EcdsaSecp256k1RecoverySignature2020": {
+      "@id": "https://w3id.org/security#EcdsaSecp256k1RecoverySignature2020"
+      "@context": {
+        "@version": 1.1,
+        "id": "@id",
+        "type": "@type",
+        "@protected": true,
+        "challenge": "https://w3id.org/security#challenge",
+        "created": {
+          "@id": "http://purl.org/dc/terms/created",
+          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+        },
+        "domain": "https://w3id.org/security#domain",
+        "expires": {
+          "@id": "https://w3id.org/security#expiration",
+          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+        },
+        "jws": "https://w3id.org/security#jws",
```

Comment:
```suggestion
        "jws": "https://w3id.org/security#jws",
```

Suggest you switch over to using multibase for the signature... unless you really want to continue using JOSE? If you continue using JOSE, the security suite will have to effectively make every non-secp256k1 parameter illegal and enforce it. I suggest that is a bad idea and will lead to larger attack surface and misimplementations (people will just pull in JOSE libraries with defaults (that support everything) and go with it).

Reply:
I've opened an issue on the specification about using proofValue and multibase:
https://github.com/decentralized-identity/EcdsaSecp256k1RecoverySignature2020/issues/12

>  If you continue using JOSE, the security suite will have to effectively make every non-secp256k1 parameter illegal and enforce it.

I read this as referring to `publicKeyJwk`, which is already defined as `@json` but the suite specification could add more guidance and/or requirements about it, so I've opened this issue:
https://github.com/decentralized-identity/EcdsaSecp256k1RecoverySignature2020/issues/13
Or did you mean about the JOSE header in the signature? I'm not sure what would be non-secp256k1 parameters there.